client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -262,69 +335,67 @@ export default defineComponent({
             >
               Open images or video to get started
             </h2>
-            <div
-              v-for="recent in paginatedRecents"
-              :key="recent.id"
-              class="pa-1"
+            <v-data-table
+              dense
+              v-bind="{ headers: headers, items: filteredRecents }"
+              :footer-props="{ itemsPerPageOptions: [10, 30, -1] }"
+              no-data-text="No data loaded"
             >
-              <h3 class="text-body-1">
+              <template #[`item.type`]="{ item }">
                 <v-icon
+                  :key="item.id"
                   class="pr-2"
                   color="primary lighten-2"
                 >
-                  {{ getTypeIcon(recent) }}
+                  {{ getTypeIcon(item) }}
```

Comment:
Do you think it's worth adding a hover tooltip saying the dataset type? Just throwing the idea out there, might be something too extra so feel free to ignore.

Reply:
I like that because the icons are kind of ????? for representation.